BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java

```diff
@@ -36,8 +39,7 @@ FieldState getMastFieldState(Coordinate coordinate) throws NoMastAtPositionExcep
         MastState mastState = getMast(coordinate).getMastState();
         if (mastState.equals(MastState.HIT)) {
```

Comment:
return mastState.equals(MastState.HIT) ? FieldState.HIT_MAST : FieldState.NOT_HIT_MAST;
 ????????? It's IMO only